frame/evm-system/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "pallet-evm-system"
+version = "1.0.0-dev"
+license = "Apache-2.0"
+description = "FRAME EVM SYSTEM pallet."
+authors = { workspace = true }
+edition = { workspace = true }
+repository = { workspace = true }
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+log = { version = "0.4.17", default-features = false }
```

Comment:
Should probably use workspace dep here as well

Reply:
https://github.com/humanode-network/frontier/pull/76/commits/96dbde2bbb3adfac8c82d7b5f298b9085bcaf81c